.github/ISSUE_TEMPLATE/2-new-feature.md

```diff
@@ -0,0 +1,22 @@
+---
+name: "\U0001F680 New Feature"
+about: Spec out a new feature
+
+---
+
+## Problem
+
+<!-- Describe the problem you are trying to solve. -->
+
+## Solution/implementation plan
+
+<!-- Describe your proposed solution. -->
+
+## Why do we need this?
```

Comment:
Seems like this is the same as `Problem`, no? Unless we rename this comletely to 'alternatives'

Reply:
It _is_ a bit redundant, but my intend was to state the problem out loud but also have a "self-accountable" space where you _do_ argument why we need to do things in a certain way. In other terms, make sure the problem is not just in your head 😁

Reply:
b7490b0